packages/@expo/server/src/runtime/error.ts

```diff
@@ -0,0 +1,76 @@
+export class StatusError extends Error {
+  status: number;
+  body: string;
+
+  constructor(status?: number, body?: { error?: string; [key: string]: any } | Error | string);
+  constructor(status?: number, errorOptions?: { cause: unknown; error?: string });
```

Comment:
The `error` property of the options is unused.

````suggestion
  constructor(status?: number, errorOptions?: { cause: unknown });
````